library/ix-dev/community/netbootxyz/templates/_netboot.tpl

```diff
@@ -0,0 +1,54 @@
+{{- define "netboot.workload" -}}
+workload:
+  netboot:
+    enabled: true
+    primary: true
+    type: Deployment
+    podSpec:
+      hostNetwork: {{ .Values.netbootNetwork.hostNetwork }}
+      containers:
+        netboot:
+          enabled: true
+          primary: true
+          imageSelector: image
+          securityContext:
+            runAsNonRoot: false
+            runAsUser: 0
+            runAsGroup: 0
+            readOnlyRootFilesystem: false
+            capabilities:
+              add:
+                - NET_BIND_SERVICE
+                - SETGID
+                - SETUID
+                - SYS_CHROOT
+          env:
+            NGINX_PORT: {{ .Values.netbootConfig.NGINX_PORT }}
+            TFTPD_OPTS: {{ .Values.netbootConfig.TFTPD_OPTS }}
```

Comment:
```suggestion
            TFTPD_OPTS: {{ join " " .Values.netbootConfig.TFTPD_OPTS }}
```

I suppose this is a space separated list of args?
Also convert the questions.yaml and values.yaml to a list

And please change the variable name to `tftpdOpts`